2024/political-donations-by-party/index.html

```diff
@@ -1,5 +1,22 @@
 <html lang="en">
     <head>
+        <style>
+            #grid-container {
+                display: inline-block;
+                position: relative;
+                width: 100%;
+                padding-bottom: 100%;
+                vertical-align: top;
+                overflow: hidden;
+            }
+            .d3-content {
+                display: inline-block;
+                position: absolute;
+                top: 0;
+                left: 0;
+            }
+        </style>
```

Comment:
https://chartio.com/resources/tutorials/how-to-resize-an-svg-when-the-window-is-resized-in-d3-js/